setup.py

```diff
@@ -52,11 +39,18 @@ def get_ucp_version():
     ),
 ]
 
-cmdclass = dict(build_ext=build_ext)
-cmdclass = versioneer.get_cmdclass(cmdclass)
+cmdclass = dict()
+cmdclass.update(versioneer.get_cmdclass())
+cmdclass["build_ext"] = build_ext
 
 setup(
+    # TODO: At present the ucx-py naming scheme will not match the rest of the
+    # RAPIDS wheels. It is hardcoded in pyproject.toml, so overriding it here
+    # will not work and wheels will always just be called ucx_py.
+    # name="ucx-py" + os.getenv("RAPIDS_PY_WHEEL_CUDA_SUFFIX", default=""),
     ext_modules=ext_modules,
     cmdclass=cmdclass,
-    version=versioneer.get_version(),
+    version=os.getenv(
+        "RAPIDS_PY_WHEEL_VERSIONEER_OVERRIDE", default=versioneer.get_version()
+    ),
```

Comment:
Maybe we should check this environment variable above and use it (skipping versioneer and skipping `cmdclass` updates)

Reply:
I've switched to the versioneer patch that we're now using in all the libraries to ensure that we get a consistent version in nightlies as well. Let me know if that addresses your comment here sufficiently or if you also want to try to avoid cmdclass changes (I don't think I'm doing special for the cmdclass when building wheels anymore, that code is identical to before now that I've undone the above change).